### PR TITLE
Chore: Removed all Dapr related dependencies

### PR DESCRIPTION
**Pull request by rajeshde**
- Updated Dockerfile assets by removing dapr dependencies and exposed the container at `9000` instead of `3500`
- Removed all @dapr/dapr dependencies
- Replaced Dapr invocation with axios call
- Made port dynamic in proxy instance and updated proxy path after removal of Dapr
- Fixed all `APP_PORT` to `9000`
- Updated package version to `0.0.54`

### CODE REVIEW

File: packages/cli/template/router.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ module.exports = () => {
     {
       "path": "/backend/engine/(.*)",
       "proxy": {
-        "instance": "engine:3500",
-        "path": "/v1.0/invoke/engine/method/$1"
+        "instance": `engine:${process.env.APP_PORT}`,
```
Comment (faizahmedfarooqui) [inline packages/cli/template/router.js]: @rajeshde Please make it `${process.env.APP_PORT || 9000}`

File: packages/cli/template/src/handlers/actions.js [hunk 2]
```diff
@@ -26,22 +26,16 @@ module.exports = async (req, res) => {
...
-    const data = await client.invoker.invoke(
-      serviceAppId,
-      serviceMethod,
-      HttpMethod.POST,
-      { ...body },
-      { headers }
-    );
+    const { data } = await axios({
+      method: req.method,
+      url: `http://${serviceAppId}:${process.env.APP_PORT}/${serviceMethod}`,
```
Comment (faizahmedfarooqui) [inline packages/cli/template/src/handlers/actions.js]: @rajeshde Remove `APP_PORT` var, and make it always to call 9000.

File: packages/cli/template/src/handlers/invoke.client.js [hunk 3]
```diff
@@ -20,20 +20,19 @@ module.exports = async (req, res) => {
...
     data = body.hasOwnProperty('data') ? { ...body.data } : {};
   }
 
   try {
-    await client.invoker.invoke(
-      appId,
-      methodName,
+    await axios({
       method,
+      url: `http://${appId}:${process.env.APP_PORT}/${methodName}`,
```
Comment (faizahmedfarooqui) [inline packages/cli/template/src/handlers/invoke.client.js]: @rajeshde Same comment, always make call to 9000 not APP_PORT.

File: packages/cli/template/src/handlers/invoke.server.js [hunk 4]
```diff
@@ -16,34 +16,28 @@ module.exports = async (req, res) => {
...
     data = body.hasOwnProperty('data') ? { ...body.data } : {};
   }
 
   try {
-    const resp = await client.invoker.invoke(
-      appId,
-      methodName,
+    const resp = await axios({
       method,
+      url: `http://${appId}:${process.env.APP_PORT}/${methodName}`,
```
Comment (faizahmedfarooqui) [inline packages/cli/template/src/handlers/invoke.server.js]: @rajeshde 9000 instead of APP_PORT

**Review by faizahmedfarooqui — CHANGES_REQUESTED** [review]

**Comment by faizahmedfarooqui** [comment]
@rajeshde One more thing, I missed out on pointing to upgrade package's version and creating new build with the newer version. Please the same changes and we're good to publish it to npm.

**Comment by rajeshde** [comment]
@faizahmedfarooqui I have fixed all `APP_PORT` to `9000`. Updated package version and created a new build.

**Review by faizahmedfarooqui — APPROVED** [review]

**Pull request merged**